tests/upload_test.go

```diff
@@ -1027,10 +1028,12 @@ var _ = Describe("Preallocation", func() {
 
 		Eventually(func() string {
 			for {
-				log, err := tests.RunKubectlCommand(f, "logs", "cdi-upload-"+dvName, "-n", f.Namespace.Name)
+				log, err := tests.RunKubectlCommand(f, "logs", uploaderPodName, "-n", f.Namespace.Name)
 				if err != nil {
 					time.Sleep(1 * time.Second)
-					continue
+					if _, err = f.K8sClient.CoreV1().Pods(f.Namespace.Name).Get(context.TODO(), uploaderPodName, metav1.GetOptions{}); !k8serrors.IsNotFound(err) {
+						continue
```

Comment:
Would it work just as well to return "" on error? it will cause the Eventually timeout to be hit, but won't ever contain the substring resulting in success